nodejs/week1/avg.js

```diff
@@ -0,0 +1,11 @@
+const args = process.argv.slice(2);
+const numbers = args
+  .map((arg) => parseFloat(arg))
+  .filter((n) => !isNaN(n) && n !== "");
```

Comment:
Good check, but I think that checking for !isNaN is enough, as empty string would be considered as not a number either way

Reply:
Silly me, true